modules/virt-configuring-storage-class-bootsource-update.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assembly:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-automatic-bootsource-updates.adoc
+//
+
+:_content-type: PROCEDURE
+[id="virt-configuring-storage-class-bootsource-update_{context}"]
+= Configuring a storage class for custom boot source updates
+
+You must configure a storage class that allows automatic updating for custom boot sources.
+
+.Procedure
+
+* Edit the HyperConverged custom resource (CR) and define a `storageClassName`:
++
+[source,yaml]
+----
+...
```

Comment:
I don't think you need `...` at the beginning of the file

Reply:
Done.